### PR TITLE
Add support for the Sepolia network

### PR DESCRIPTION
**Pull request by zah**
Implements https://github.com/status-im/nimbus-eth2/pull/3762

### CODE REVIEW

File: docs/the_nimbus_book/src/attestation-performance.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ The following options are available:
 
 Where:
 
-- The `network` can either be `mainnet`, `prater` or `ropsten`
+- The `network` can either be `mainnet`, `prater`, `ropsten` or `sepolia`
 
-- The default location of the `db`  is either `build/data/shared_mainnet_0/db`, `build/data/shared_prater_0/db` or `build/data/shared_ropsten_0/db`
+- The default location of the `db`  is `build/data/shared_mainnet_0/db` for `mainnet`, `build/data/shared_prater_0/db` for `proter`, etc.
```
Comment (etan-status) [inline docs/the_nimbus_book/src/attestation-performance.md]:
```suggestion
- The default location of the `db`  is `build/data/shared_mainnet_0/db` for `mainnet`, `build/data/shared_prater_0/db` for `prater`, etc.
```

File: run-sepolia-beacon-node.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2020-2021 Status Research & Development GmbH. Licensed under
```
Comment (etan-status) [inline run-sepolia-beacon-node.sh]:
```suggestion
# Copyright (c) 2022 Status Research & Development GmbH. Licensed under
```

File: beacon_chain/networking/network_metadata.nim [hunk 3]
```diff
@@ -271,6 +273,8 @@ when not defined(gnosisChainBinary):
           praterMetadata
         of "ropsten":
           ropstenMetadata
+        of "sepolia":
+          sepoliaMetadata
```
Comment (etan-status) [inline beacon_chain/networking/network_metadata.nim:277]: `configDefaults` seems to already have an entry for `sepolia`. I assume that one is still correct

**Review by etan-status — APPROVED** [review]

File: beacon_chain/networking/network_metadata.nim (hunk 3 above)
Comment (zah) [reply beacon_chain/networking/network_metadata.nim:277]: Can you clarify what you mean? Do you refer to the `shouldSupportLightClient` value?

Comment (etan-status) [reply beacon_chain/networking/network_metadata.nim:277]: Yep.

```
          let data = (genesisData[0 ..< 40].toHex())
          data in [
            # Kiln
            "0C572B620000000099B09FCD43E5905236C370F184056BEC6E6638CFC31A323B304FC4AA789CB4AD",
            # Ropsten
            "F0DB94620000000044F1E56283CA88B35C789F7F449E52339BC1FEFE3A45913A43A6D16EDCD33CF1",
            # Prater
            "60F4596000000000043DB0D9A83813551EE2F33450D23797757D430911A9320530AD8A0EABC43EFB",
            # Sepolia
            "607DB06200000000D8EA171F3C94AEA21EBC42A1ED61052ACF3F9209C00E4EFBAADDAC09ED9B8078",
          ]
```

Wondering if the `Sepolia` value here is still accurate.

Comment (zah) [reply beacon_chain/networking/network_metadata.nim:277]: Looks correct

**Comment by zah** [comment]
Oops, I meant to enable auto-merge, but I merged the branch instead.

**Pull request merged**